### DISCUSSION TITLE
how to change https to ssh protocol in push_to_git_remote

### DISCUSSION THREAD

**Discussion by Rickkorsten** [fastlane/fastlane#17775] [Q&A]
Hello,

Is it possible to change the protocol from https to ssh in  **push_to_git_remote** ?

thanks!

**Comment by rogerluan** [comment]
Hey @Rickkorsten 👋 

That action will execute whatever is configured in your system's git config. If you have ssh set up correctly and cloned your repo using ssh, it will use ssh.

The action itself is just a command line builder: based on the arguments you pass, it will construct a `git push arg1 arg2 arg3 etc` command line and execute it.

Hope this helps.
